spec/spec_helper/mock_response/user_meta_value.data

```diff
@@ -0,0 +1 @@
+user_meta_value
```

Comment:
I think this file extension is `txt` because the response content type is "plain/text".
> https://github.com/ricohapi/media-storage-rest/blob/master/media.md#example-response-9
How about you?